Assets/MixedRealityToolkit.SDK/Experimental/HandCoach/Scripts/MoveToTarget.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Collections;
+using System.Net.NetworkInformation;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    public class MoveToTarget : MonoBehaviour
+    {
+        [Tooltip("Object to track")]
+        [SerializeField]
+        private GameObject m_trackingObject = null;
+
+        [Tooltip("Target to move to")]
+        [SerializeField]
+        private GameObject m_targetObject = null;
+
+        [Tooltip("Shared parent between tracking and target objects used for relative local positions")]
+        [SerializeField]
+        private GameObject m_rootObject = null;
+
+        [Tooltip("Duration of move from tracking object to target object")]
+        [SerializeField]
+        private float m_duration = 1.38f;
+
+        [Tooltip("Tunable offset to get the GameObject to arrive at the right target position")]
+        [SerializeField]
+        private Vector3 m_targetOffset = new Vector3(0.05f, -0.1f, -0.2f);
+
+        [Tooltip("Lerp curve")]
+        [SerializeField]
+        private AnimationCurve m_animationCurve = AnimationCurve.Linear(0f, 0f, 1f, 1f);
+
+        // The local position of this gameObject
+        private Vector3 m_position;
+
+        // The local position of m_targetObject
+        private Vector3 m_targetPosition;
+
+        // The offset from this gameObject's position to the m_trackingObject's position
+        private Vector3 m_trackingOffset;
```

Comment:
As in previous file (and everywhere else), please don't use hungarian notation, to ensure consistentcy with rest of MRTK code.